### ISSUE TITLE
Not handling scenario without ORIN

### ISSUE THREAD

**Issue by remintz** [time4tea/gopro-dashboard-overlay#60]
I'm running version v0.53.0 and when I try to convert to csv or generate the dashboard I get the following exception:

```
Traceback (most recent call last):
  File "venv/bin/gopro-to-csv.py", line 37, in <module>
    ts = framemeta_from(args.input,
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/framemeta.py", line 239, in framemeta_from
    return parse_gopro(gpmd_from, units, metameta)
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/framemeta.py", line 230, in parse_gopro
    accl_frame_meta = accl_framemeta(gopro_meta, units, metameta=metameta)
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/framemeta.py", line 203, in accl_framemeta
    meta.accept(
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/gpmd.py", line 311, in accept
    item.accept(visitor)
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/gpmd.py", line 189, in accept
    i.accept(container_visitor)
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/gpmd.py", line 191, in accept
    container_visitor.v_end()
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/gpmd_visitors_xyz.py", line 71, in v_end
    self._on_end(
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/gpmd_visitors_xyz.py", line 147, in <lambda>
    return XYZStreamVisitor(on_end=lambda i: self._on_item(self._counter, i))
  File "/home/remintz/projects/gopro/gopro-dashboard-overlay/venv/lib/python3.8/site-packages/gopro_overlay/gpmd_visitors_xyz.py", line 115, in convert
    correct_orientation = components.orin.apply(point)
AttributeError: 'NoneType' object has no attribute 'apply'
```

Trying to trace the error I found the problem is the lack of an ORIN tag on my video. This video was recorded on a GoPro Max and treated on the GoPro Player.

I was able to run using v.0.48.0

Thanks

**Comment by time4tea** [comment]
Hi. Thanks for reporting this issue. I'll take a look. If you could share a very short video that demonstrates the problem, that would be very helpful. The email is gopro-overlay [at] time4tea.net - maybe use wetransfer?

**Comment by remintz** [comment]
Thanks. I sent using wetransfer and the above email.

**Comment by time4tea** [comment]
Thanks - just downloaded. Love the video! So exciting to see what types of activity people are using this software for. If there are any other widgets that would be useful, please don't hesitate to ask. I can't promise, but it is great to know what would be useful for people.

**Comment by time4tea** [comment]
I've opened an issue with GoPro to make sure I'm understand this correctly.

https://github.com/gopro/gpmf-parser/issues/165

**Comment by time4tea** [comment]
Probably fixed in 0.54.0 - It won't crash anymore, I'm expecting, but I'm not 100% sure the acceleration values are correct. They seem to be be, using your test video.
